### PR TITLE
PIE-27: Created contact forms

### PR DESCRIPTION
**Pull request by mrdollarbill**
Created 3 contact forms for the Contact Us page and the associated styling.
To display the forms, Contact Form 7 plugin must be installed and contact forms XML file must be imported into your installation of Wordpress.

### CODE REVIEW

File: assets/css/bootstrap/bootstrap.css [hunk 1]
```diff
@@ -400,6 +400,18 @@ h6, .h6 {
   font-size: 1rem;
 }
 
+span.wpcf7-list-item {
+  display: block;
+  margin-left: 0;
```
Comment (pritpalc) [inline assets/css/bootstrap/bootstrap.css]: Better to use the bootstrap classes for this.

File: assets/css/bootstrap/bootstrap.css [hunk 2]
```diff
@@ -400,6 +400,18 @@ h6, .h6 {
...
+}
+
+.wpcf7-free-text {
+  background: transparent;
+  border: none;
+  border-bottom: 1px solid #000000;
+  padding-top:2px;
```
Comment (pritpalc) [inline assets/css/bootstrap/bootstrap.css]: And the padding

File: assets/css/bootstrap/bootstrap.css [hunk 3]
```diff
@@ -3908,21 +3920,26 @@ input[type="button"].btn-block {
 }
 
 .nav-tabs {
-  border-bottom: 1px solid #dee2e6;
+  border-bottom: 1px solid #525252;
+  padding-right: 5%;
+  padding-left: 5%;
```
Comment (pritpalc) [inline assets/css/bootstrap/bootstrap.css]: Use the bootstrap classes here too

File: style.css [hunk 4]
```diff
@@ -26,4 +26,10 @@ Text Domain: startertheme
 .pie-footer ul, li {
   list-style: none;
   padding-left: 0;
+}
+
+
+/* CONTACT FORMS */
+.cf-holder {
+  width: 90%;
```
Comment (pritpalc) [inline style.css]: Try using a bootstrap class for this too, but if those don't work out, you can stick with 90%.

File: template-parts/contact-forms.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      <div class="tab-content">
+        <div class="tab-pane fade show active cf-holder mx-auto" id="patients-form" role="tabpanel" aria-labelledby="patients-tab">
+          <?php echo do_shortcode('[contact-form-7 id="28" title="Request for Patients"]');?>
+        </div>
+        <div class="tab-pane fade cf-holder mx-auto" id="volunteers-form" role="tabpanel" aria-labelledby="volunteers-tab">
+		  <?php echo do_shortcode('[contact-form-7 id="56" title="Request for Volunteers"]');?>
+		</div>
+        <div class="tab-pane fade cf-holder mx-auto" id="general-form" role="tabpanel" aria-labelledby="general-tab">
+		  <?php echo do_shortcode('[contact-form-7 id="57" title="General Inquiry"]');?>
+		</div>
```
Comment (pritpalc) [inline template-parts/contact-forms.php]: Fix the indenting

**Comment by pritpalc** [comment]
How would I enable or disable a contact form? Let's say I don't want to display one of the three contact forms, is that functionality included in here?

File: assets/css/bootstrap/bootstrap.css (hunk 1 above)
Comment (mrdollarbill) [reply assets/css/bootstrap/bootstrap.css]: The wpcf7-list-item spans are generated by the Contact Forms 7 plugin, and there's no way to assign them a class of ml-0 (unless we edit the CF7 plugin code). I tried assigning the ml-0 class in the contact form designer (using [checkbox class:ml-0]), but it doesn't have the intended effect.

**Comment by mrdollarbill** [comment]
> How would I enable or disable a contact form? Let's say I don't want to display one of the three contact forms, is that functionality included in here?
Added functionality to enable/disable the 3 forms individually via 3 Custom Fields.
When all 3 forms are disabled, just an "Email Us" link will be displayed on the Contact Us page.

**Pull request merged**